rallyrolebot/cogs/rally_cog.py

```diff
@@ -41,31 +42,20 @@ async def set_rally_id(self, ctx, rally_id):
         data.add_discord_rally_mapping(ctx.author.id, rally_id)
 
     @commands.command(name="price", help="Get the price data of a coin")
-    @validation.is_valid_coin()
-    async def price(self, ctx, coin_symbol):
-        def increase_decrease_gradient_index(percentage):
-            gradient = 5
-            if isinstance(percentage, float) and percentage != 0:
-                rank = math.floor(percentage / 20)
-                if(rank >= 5):
-                    rank = 5
-                elif(rank < -5):
-                    rank = -5
-                elif rank >= 0:
-                    rank += 1
-                gradient += rank
-            return gradient
+    async def price(self, ctx, coin_symbol : Union[CreatorCoin, CommonCoin]):
+        def get_gradient_color(percentage):
+            if percentage < 0:
+                return get_gradient_by_percentage(str(WHITE_COLOR), str(DARK_RED_COLOR), PRICE_GRADIENT_DEPTH, abs(percentage))
+            else:
+                return get_gradient_by_percentage(str(WHITE_COLOR), str(DARK_GREEN_COLOR), PRICE_GRADIENT_DEPTH, percentage)
```

Comment:
Ah seems like the if-else is unnecessary (just call `abs` :-) )

Reply:
Color also changes depending on the percentage(DARK_RED_COLOR for negative, DARK_GREEN_COLOR for positive).